print-numbers-server/index.js

```diff
@@ -0,0 +1,23 @@
+//Call the http module and define several constants (http,host,port) 
+const http = require("http");
+const host = "127.0.0.1";
+const port = 8000;
+let stringNumbers = "";
+
+const requestListener = function(req, res) {
+  const responseNumbers = print_numbers();
+  res.end(responseNumbers);
+};
+
+const server = http.createServer(requestListener);
+server.listen(port, host, () => {
+  console.log("Reader", port);
+  console.log(`Server is running on http://${host}:${port}`);
+});
+
+function print_numbers() {
+  for (let i = 0; i <= 100; i++) {

```

Comment:
what about to use Array object props? Array.from({ length: 100}).map(() => -...)

Reply:
Done